beets/util/queue.py

```diff
@@ -164,13 +170,15 @@ def get(self):
             while True:
                 if self._invalidated:
                     return self._invalid_value
-                elif not len(self._queue):
-                    if self._finished:
-                        self._invalidate(self._on_finished)
-                        return self._on_finished
-                    else:
-                        self.not_empty.wait()
-                else:
+
+                if len(self._queue):
```

Comment:
Not sure about this, but could this be `if self._queue` (the same way lists work)?

Reply:
Yep, that is certainly more pythonic!